.circleci/config.yml

```diff
@@ -610,11 +610,11 @@ jobs:
                 ;;
               "test")
                 echo 'export TEST_COLLABORATOR=$PUPPETEER_COLLABORATOR_TEST' >> $BASH_ENV
-                echo 'export USER_NAME=$(eval echo "\$PUPPETEER_USER_TEST_${CIRCLE_NODE_INDEX}@fake-research-aou.org")' >> $BASH_ENV
+                echo 'export USER_NAME=$(eval echo "\$PUPPETEER_USER_TEST@fake-research-aou.org")' >> $BASH_ENV
```

Comment:
nit: `eval echo` should no longer needed here, can just do a simple interpolation now

Reply:
👍